ontology/rml-core.owl

```diff
@@ -0,0 +1,517 @@
+#    Annotation properties
+#################################################################
+
+###  http://creativecommons.org/ns#license
+cc:license rdf:type owl:AnnotationProperty .
+
+
+###  http://purl.org/dc/elements/1.1/contributor
+dc:contributor rdf:type owl:AnnotationProperty .
+
+
+###  http://purl.org/dc/elements/1.1/creator
+dc:creator rdf:type owl:AnnotationProperty .
+
+
+###  http://purl.org/dc/elements/1.1/description
+dc:description rdf:type owl:AnnotationProperty .
+
+
+###  http://purl.org/dc/elements/1.1/issued
+dc:issued rdf:type owl:AnnotationProperty .
+
+
+###  http://purl.org/dc/elements/1.1/modified
+dc:modified rdf:type owl:AnnotationProperty .
+
+
+###  http://purl.org/dc/elements/1.1/publisher
+dc:publisher rdf:type owl:AnnotationProperty .
+
+
+###  http://purl.org/dc/elements/1.1/title
+dc:title rdf:type owl:AnnotationProperty .
+
+
+###  http://purl.org/vocab/vann/preferredNamespacePrefix
+vann:preferredNamespacePrefix rdf:type owl:AnnotationProperty .
+
+
+###  http://purl.org/vocab/vann/preferredNamespaceUri
+vann:preferredNamespaceUri rdf:type owl:AnnotationProperty .
+
+
+###  http://www.linkedmodel.org/schema/vaem#acronym
+vaem:acronym rdf:type owl:AnnotationProperty .
+
+
+###  http://xmlns.com/foaf/0.1/name
+foaf:name rdf:type owl:AnnotationProperty .
+
+
+#################################################################
+#    Object Properties
+#################################################################
+
+###  http://w3id.org/rml/class
+:class rdf:type owl:ObjectProperty ;
+       rdfs:domain :SubjectMap ;
+       rdfs:range rdfs:Class ;
+       rdfs:comment "The subject value generated for a logical source record will be asserted as an instance of this RDFS class."@en ;
+       rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+       rdfs:label "class" .
+
+
+###  http://w3id.org/rml/constant
+:constant rdf:type owl:ObjectProperty ;
+          rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+          rdfs:label "constant" .
+
+
+###  http://w3id.org/rml/datatype
+:datatype rdf:type owl:ObjectProperty ;
+          rdfs:domain :ObjectMap ;
+          rdfs:range rdfs:Datatype ;
+          rdfs:comment "Specifies the datatype of the object component for the generated triple from a logical source record."@en ;
+          rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+          rdfs:label "datatype" .
+
+
+###  http://w3id.org/rml/datatypeMap
+:datatypeMap rdf:type owl:ObjectProperty ;
+             rdfs:isDefinedBy <http://w3id.org/rml/core/> ;
+             rdfs:label "datatype map" .
+
+
+###  http://w3id.org/rml/graph
+:graph rdf:type owl:ObjectProperty ;
+       rdfs:domain [ rdf:type owl:Class ;
+                     owl:unionOf ( :PredicateObjectMap
+                                   :SubjectMap
+                                 )
+                   ] ;
+       rdfs:comment "An IRI reference for use as the graph name of all triples generated with the GraphMap."@en ;
```

Comment:
"all triples generated with the GraphMap": there is no graph map here, right? Should be 
"all triples generated with the term map".